docker/1.15.2/py37/Dockerfile.cpu

```diff
@@ -113,11 +113,8 @@ RUN ${PIP} install --no-cache-dir -U \
     pandas==0.24.2 \
     Pillow==7.0.0 \
     h5py==2.10.0 \
-    keras_applications==1.0.8 \
-    keras_preprocessing==1.1.0 \
     requests==2.22.0 \
     smdebug==0.7.2 \
-    sagemaker==1.56.1 \
```

Comment:
Why are we removing this package?

Reply:
sagemaker-python sdk doesnt support py3.7 platform so yuan said we should remove the package as well so that customer knows the features are not supported

Reply:
Let's make sure this is documented in release notes.